moj-node/server/services/search.js

```diff
@@ -1,10 +1,17 @@
-module.exports = function createSearchService({ searchRepository }) {
-  function find({ query, limit, from }) {
-    return searchRepository.find({ query, limit, from });
+const utils = require('../utils');
+
+module.exports = function createSearchService({
+  searchRepository,
+  getEstablishmentName = utils.getEstablishmentName,
+}) {
+  function find({ query, limit, from, establishmentId }) {
```

Comment:
let's add a check to see if we have a query here?

Reply:
Resolved, added assertion for query string